packages/account/src/extensions/src9/src9.cairo

```diff
@@ -142,5 +135,26 @@ pub mod SRC9Component {
             let mut src5_component = get_dep_component_mut!(ref self, SRC5);
             src5_component.register_interface(interface::ISRC9_V2_ID);
         }
+
+        /// Validates that a signature is valid for a given hash and account.
+        /// Checks if the signature is valid according to SRC6 standard.
+        /// Supports both the VALIDATED constant and the legacy "1" boolean value.
+        ///
+        /// Revert if the signature is invalid for the given hash and account
+        fn assert_valid_signature(
```

Comment:
The idea of the issue is to move this helper into account utils, and reuse it, instead of having to define it in every place where is used. Note that this helper doesn't require self (is not used) so can be defined without a dependency on the component.